### DISCUSSION TITLE
Adonis Mail Custom Timeouts

### DISCUSSION THREAD

**Discussion by JKolanko** [adonisjs/core#4715] [Help]
Hey there,

Wondering if there is any way to set a custom timeout value in Adonis mail like you can with nodemailer?  Some of the default settings are way too high for us, such as waiting 2 minutes for a connection, and we wanted to adjust those. I did not see any mention of this in the mail documentation.

Link to nodemailer documentation: https://nodemailer.com/smtp/#connection-options

Thanks!

**Comment by thetutlage** [comment]
Yeah, it seems like we will have to expose these options.
